UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -396,7 +403,8 @@ public static void Play(string name, AudioSourceParameters audioSourceParameters
 
 		ApplyAudioSourceParameters(audioSourceParameters, sound.audioSource);
 
-		Instance.PlaySource(sound, polyphonic, true, audioSourceParameters != null && audioSourceParameters.MixerType != MixerType.Unspecified);
+		Instance.PlaySource(sound, polyphonic, true,
```

Comment:
Codacy found an issue: [Remove this default value assigned to parameter 'Global'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)